test/data/generate_dictionary_test_epub.py

```
@@ -0,0 +1,139 @@
+    z.writestr("OEBPS/content.opf", opf(CHAPTERS), compress_type=zipfile.ZIP_DEFLATED)
+    z.writestr("OEBPS/toc.ncx", ncx(CHAPTERS), compress_type=zipfile.ZIP_DEFLATED)
+    z.writestr("OEBPS/cover.xhtml", xhtml("Dictionary Feature Tests", COVER_BODY),
+               compress_type=zipfile.ZIP_DEFLATED)
+    z.writestr("OEBPS/toc-notice.xhtml", xhtml("Table of Contents", TOC_NOTICE_BODY),
+               compress_type=zipfile.ZIP_DEFLATED)
+    for i, (title, body) in enumerate(CHAPTERS):
+        z.writestr(f"OEBPS/ch{i+1}.xhtml", xhtml(title, body),
+                   compress_type=zipfile.ZIP_DEFLATED)
+
+print(f"Written: {OUT}")
+print(f"Front matter: cover.xhtml, toc-notice.xhtml (spine only, not in chapter browser)")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)